pixart.js

```diff
@@ -1 +1,25 @@
+var button = document.getElementById("set-color");
+var userInput = document.getElementById("color-field");
 
+var brush = document.querySelector("brush");
+
+var setColor = function(){
+  button.addEventListener("click", function(event){
+    event.preventDefault(); //googled this - not sure exactly what it is doing still.
```

Comment:
Buttons have a default action of refreshing the page, same as `<a>` has a default action of going to a new page. This prevents that from happening.